coalib/bears/meta.py

```diff
@@ -22,4 +24,6 @@ def __init__(cls, clsname, bases, clsattrs, *varargs, aspects=None):
         """
         type.__init__(cls, clsname, bases, clsattrs, *varargs)
         if aspects is not None:
-            cls.aspects = defaultdict(lambda: [], aspects)
+            cls.aspects = defaultdict(
+                lambda: aspectlist([]),
+                {k: aspectlist(v) for (k, v) in dict(aspects).items()})
```

Comment:
Don't get angry with me please ;) but using a pair generator has better performance than creating another intermediate `dict`:

```
((k, aspectlist(v)) for (k, v) in dict(aspects).items())
```

But it looks not so nice :/ Hmm... it's more important that aspects code looks nice! :D